nbgitpuller/hookspecs.py

```diff
@@ -0,0 +1,22 @@
+import pluggy
```

Comment:
I think we should name this file plugin_hook_specs.py or similar. I really appreciate having it clearly tied to plugins at least.